source/features/latest-tag-button.tsx

```diff
@@ -18,45 +30,83 @@ const getLatestTag = cache.function(async (): Promise<string | false> => {
 			}) {
 				nodes {
 					name
+					target {
+						oid
+						... on Commit {
+							committedDate
+						}
+					}
 				}
 			}
 		}
 	`);
 
-	const tags: string[] = repository.refs.nodes.map((tag: {name: string}) => tag.name);
+	const tags: TagInfo[] = repository.refs.nodes.map((tag: {name: string; target: {oid: string; committedDate: string}}) => new TagInfo(tag));
 	if (tags.length === 0) {
 		return false;
 	}
 
 	// If all tags are plain versions, parse them
-	if (tags.every(tag => /^[vr]?\d/.test(tag))) {
-		return tags.sort(compareVersions).pop()!;
+	if (tags.every(tag => /^[vr]?\d/.test(tag.name))) {
+		return tags.sort((o1, o2) => compareVersions(o1.name, o2.name)).pop()!;
 	}
 
 	// Otherwise just use the latest
 	return tags[0];
 }, {
 	expiration: 1,
-	cacheKey: () => __featureName__ + ':' + getRepoURL()
+	cacheKey: () => __featureName__ + '_tags:' + getRepoURL()
+});
+
+const masterHasCommitsAfter = cache.function(async (commitDate: string): Promise<boolean> => {
+	// Find out whether we have commits after a certain date
+	const repository = await api.v4(`
+		repository(${getRepoGQL()}) {
+			object(expression: "master") {
```

Comment:
It’s not `master` but the default branch. Use `defaultBranchRef` if possible, or our own `getDefaultBranch()` function